src/cryptography/hazmat/primitives/asymmetric/ec.py

```diff
@@ -11,6 +11,58 @@
 from cryptography import utils
 
 
+class EllipticCurveField(object):
+    """
+    Field type of an EllipticCurve
+    """
+
+
+class EllipticCurvePrimeField(EllipticCurveField):
```

Comment:
The field() method on the EllipticCurve class returns an EllipticCurveField (either a prime field or a binary field). This parent class is intended to show that relationship. At the moment, there are no common methods that requires the child classes to inherit from it, but I think it's still good from a documentation standpoint to see the relationship here and have a type name I can include in the public docs for the return type of field().

For the curves, I changed them to inherit from EllipticCurve because in the full version of my check-in there are common methods now implemented on EllipticCurve for point encoding/decoding that all the curves share (moved out of EllipticCurvePublicNumbers because I need them for the base_point() method).